### ISSUE TITLE
Unable to set arbitrary environment values

### ISSUE THREAD

**Issue by zavyrylin** [funbox/init-exporter#20]
A attempt to export simple procfile v2 with command section like the following:
```yaml
commands:
  app1:
    env:
      JAVA_OPTS: '"-Xms512m -Xmx1g -XX:+HeapDumpOnIutOfMemoryError"'
      LOADER_SYSTEM: true
    command: bin/run-something
```
leads to export failure with error message:
```
Errors while application validation:
- Environment variable value "-Xms512m -Xmx1g -XX:+HeapDumpOnOutOfMemoryError" is insecure and can't be accepted
```

I believe it is quite common to setup some configuration through environment variables defined that way, so it seems that validation rules applied on export are too strict.

**Comment by zavyrylin** [comment]
I humbly considered a couple of obvious solutions might be used. Here are they:
* disable security validations (of some kind) completely;
* disable (some) security validations by default, but enable them by command argument;
* fix validation rules.

Disabling validation completely is obvious not an option since security is always an issue and should never be neglected.

Disabling/enabling by command line argument is more desirable, so that enabling by default is a breaking change possibly ruining existing deployments, and in my opinion negative variant (disabled by default) is more appropriate.

Fixing validation rules might become a solution but it has its own drawback: all the rules possible for all the cases will never be taken into account entirely, so than failing a deployment is a matter of time. In that sense with this option we will shoot ourself in the foot later.

**Comment by andyone** [comment]
Fixed in #21
